ush/config_defaults.yaml

```diff
@@ -886,6 +895,9 @@ workflow:
 
   CYCL_HRS_SPINSTART: []
   CYCL_HRS_PRODSTART: []
+  BOUNDARY_LEN_HRS: 0
```

Comment:
Should this not default to `FCST_LEN_HRS`? Zero does not seem consistent with the description above.

Reply:
BOUNDARY_LEN_HRS can be longer than FCST_LEN_HRS because RRFS pre-fetches boundary conditions in advance not only for the current cycle forecast lenght. I think I asked this question a while ago and MIng clarified it like that. Here is the relevant code in make_lbcs
```
  end_hr=$FCST_LEN_HRS
  if [ $BOUNDARY_LEN_HRS -gt $end_hr ]; then
     end_hr=$BOUNDARY_LEN_HRS
  fi
```

Reply:
I see now you are talking about the default value, not that it should be set to `FCST_LEN_HRS`.
RRFS_dev1 sets both variables to default to 0
https://github.com/NOAA-GSL/regional_workflow/blob/feature/RRFS_dev1/ush/config_defaults.sh#L581-L582
The loops in the workflow that use these variables will behave differently if defaulted to `FCST_LEN_HRS`.
Can we just leave things as they are as long as it is consistent with RRFS_dev1 atleast for now?  We can
thinker with things later once we have a setup to test things..